.travis.yml

```diff
@@ -13,12 +13,14 @@ addons:
 
 before_install:
     - export CXX="g++-4.8"
-    - npm install juttle@^0.4.0
 
 node_js:
     - '4.2'
-    - '5.0'
 
+before_script:
+    - npm install juttle/juttle;
+    - pushd node_modules/juttle; npm install; popd
```

Comment:
@demmer is this the way to handle installing top of juttle master ? or should we simply do `npm install juttle` for the latest juttle ?